### PR TITLE
Various frontend improvements

### PR DESCRIPTION
**Pull request by rd3k**
Optimise shadow image (25.7% filesize reduction), fix chmod in README, HTML5 doctype (why was it "loose"?), make all JS external, concatenate and minified.

### CODE REVIEW

**Comment by Pezmc** [comment]
Looks great to me, not sure why it was loose. I did the frontend in a rush perhaps!

File: index.html [hunk 1]
```diff
@@ -1,10 +1,9 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!DOCTYPE html>
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <title>CS Timetable to iCal</title>
 <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
-<script src="js/jquery.steps.min.js"></script>
```
Comment (Pezmc) [inline index.html:7]: Where was this moved to? (Reviewing on a phone)

Comment (rd3k) [inline index.html:7]: It's been concatenated with main.js into main.min.js. I've got this automated in CodeKit, but perhaps there should be a "build script" for if main.js ever changes.

Comment (Pezmc) [inline index.html:7]: That's probably a good idea!

## 

07845 594224
Sent from a mobile device

File: templates/page4.twig [hunk 2]
```diff
@@ -1,6 +1,10 @@
+<input id="tickall" type="button" class="clean-gray" value="Tick all"/>
+<input id="clear" type="button" class="clean-gray" value="Clear"/>
```
Comment (rd3k) [inline templates/page4.twig:2]: Good idea?

Comment (Pezmc) [inline templates/page4.twig:2]: Yep great! Can't merge till I have my laptop though

**Pull request merged**